### PR TITLE
Add promotion actions and promotion calculators documentation for end users

### PR DESCRIPTION
**Pull request by benjaminwil**
This adds documentation about Solidus's promotion actions and calculators for end users. The audience for this documentation is store administrators or anyone who has to touch the solidus_backend as a business person.

### CODE REVIEW

**Review by kennyadsl — APPROVED** [review]
Thanks!

File: guides/source/users/promotions/promotion-calculators.html.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+# Promotion calculators
+
+When you create a promotion action, one step is to select the promotion
+calculator that it should use. You can do this by selecting a value in the
+**Calculated By** drop-down menu.
+
+Depending on the [type of promotion action][promotion-action-types] that you are
```
Comment (tvdeyen) [inline guides/source/users/promotions/promotion-calculators.html.md]: missing link

File: guides/source/users/promotions/promotion-calculators.html.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Promotion calculators
+
+When you create a promotion action, one step is to select the promotion
```
Comment (tvdeyen) [inline guides/source/users/promotions/promotion-calculators.html.md]: Maybe add a short sentence what a calculator does. It's often not clear for non technical people what this thing is for, besides that it 'calculates things'.

File: guides/source/users/promotions/promotion-calculators.html.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+calculator that it should use. You can do this by selecting a value in the
+**Calculated By** drop-down menu.
+
+Depending on the [type of promotion action][promotion-action-types] that you are
+creating, only a subset of the promotion calculators may be available to use.
+
+This table outlines the outlines the available promotions calculators and which
+promotion action types they can be applied to:
```
Comment (tvdeyen) [inline guides/source/users/promotions/promotion-calculators.html.md]: The header of this table is misleading. There are no promotion action types listed, these are calculables (a term we probably should explain as well).

**Review by tvdeyen — CHANGES_REQUESTED** [review]
The promotion calculators table header should be improved. Other than that 👍

**Comment by benjaminwil** [comment]
@tvdeyen Thank you for the feedback. I think my most recent commit addresses all your comments.

**Review by tvdeyen — COMMENTED** [review]
One small typo. But this good to go then

**Review by tvdeyen — APPROVED** [review]

**Pull request merged**